test/pluggy_elixir/http_client_test.exs

```diff
@@ -1,11 +1,11 @@
 defmodule PluggyElixir.HttpClientTest do
-  use PluggyElixir.Case, async: true
+  use PluggyElixir.Case, async: false
```

Comment:
Do we really need to be sync in here?

Reply:
Yes, because we cant run async tests with `Bypass` before we have merged the #10